src/helpers/component-tree.ts

```diff
@@ -92,3 +92,22 @@ export function getHostSiblings(
     (sibling) => !hostSelves.includes(sibling)
   );
 }
+
+/**
+ * Returns the unsafe root element of the tree (probably composite).
+ *
+ * @param element The element start traversing from.
+ * @returns The root element of the tree (host or composite).
+ */
+export function getRootElement(element: ReactTestInstance | null) {
```

Comment:
if it's unsafe, maybe let's state this explicitly with `getHostOrCompositeRootElement` or `unsafe_getRootElement`?

Reply:
it's internal fn, none of functions in this file are exported publically, so it's safe for us ;-)
Anyway I'll go for `getUnsafeRootElement()`

Reply:
I believe it should raise awareness in contributors as well :)